README.md

```diff
@@ -6,6 +6,20 @@ SageMaker TensorFlow Serving Container is an a open source project that builds
 docker images for running TensorFlow Serving on 
 [Amazon SageMaker](https://aws.amazon.com/documentation/sagemaker/).
 
+Supported versions of TensorFlow: ``1.4.1``, ``1.5.0``, ``1.6.0``, ``1.7.0``, ``1.8.0``, ``1.9.0``, ``1.10.0``, ``1.11.0``, ``1.12.0``, ``1.13.1``, ``1.14.0``, ``1.15.0``, ``2.0.0``.
+
+Supported versions of TensorFlow for Elastic Inference: ``1.11.0``, ``1.12.0``, ``1.13.1``, ``1.14.0``.
+
+ECR repositories for SageMaker built TensorFlow Serving Container:
+
+- `'tensorflow-inference'` in `763104351884` AWS account for any new version starting with ``1.13.0``;
```

Comment:
This account is for region without opt-in restriction (non HKG/BAH), I'd suggest either mention all these accounts, or none of them to reduce confusion. Same for below account.